tutorials/3d/lighting.rst

```diff
@@ -106,6 +117,6 @@ Baked light
 -----------
 
 Baked light stands for pre-computed ambient light. It can serve multiple
-purposes, such as baking light emissors that are not going to be used in
+purposes, such as baking light emitters that are not going to be used in
 real-time, and baking light bounces from real-time lights to add more
 realism to a scene (see Baked Light]] tutorial for more information).
```

Comment:
Those `]]` are a leftover from a previous wiki-style link. Maybe you could also fix them to point to the Baked Lights tutorial? (<code>:ref:\`doc_light_baking\`</code>)